testing/cuda/CMakeLists.txt

```
@@ -40,3 +40,8 @@ add_cmake_config_test( set_arch-existing.cmake )
 add_cmake_config_test( set_arch-invalid-mode.cmake )
 add_cmake_config_test( set_arch-native.cmake )
 add_cmake_config_test( set_arch-rapids.cmake )
+
+set(deprecateds_message "rapids-cmake policy [deprecated=23.02 removed=23.06]: Usage of `ALL`")
```

Does the “s” mean something in “deprecateds”?
```suggestion
set(deprecated_message "rapids-cmake policy [deprecated=23.02 removed=23.06]: Usage of `ALL`")
```

---

Corrected